pkgs/tools/system/glances/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, fetchFromGitHub, fetchpatch, python3 }:
+
+with python3.pkgs;
+
+buildPythonApplication rec {
+  pname = "glances";
+  version = "3.1.3";
+
+  src = fetchFromGitHub {
+    owner = "nicolargo";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "15yz8sbw3k3n0729g2zcwsxc5iyhkyrhqza6fnipxxpsskwgqbwp";
+  };
+
+  # fix unitest.py
+  patches = [
+    ( fetchpatch {
```

Comment:
looks like a pretty small patch. is there a policy to prefer referencing patches like this, instead of including them directly?

Reply:
it has been merged into their master already, which is why I prefer the fetchpatch.

Also communicates that I don't need to upstream the fix

Reply:
ah, makes sense. maybe link to the upstream issue and mention it can be dropped on the next version bump? :)

Reply:
correct, oddly enough, this patch was made the same day as the last version. So I'm not sure if he forgot to include it in the release. Either way, it should be dropped next release, I'll make a note of it